hi，I installed 1.0.1+2, but something went wrong?

`[!] No podspec found for `livechatt` in `.symlinks/plugins/livechatt/ios``

I tried to install 1.0.1+1 and got the same result. Is this a scan code problem?

@Mastersam07 hi,I have installed "**livechatt 1.0.1+3**", but it still does not work?

`livechatt: ^1.0.1+3`

`/Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator14.0.sdk/System/Library/Frameworks/UIKit.framework/Headers/UIWindowScene.h:17:50: note: 'UIWindowScene' has been marked as being introduced in iOS 13.0 here, but the deployment target is iOS 9.0.0
    UIKIT_EXTERN API_AVAILABLE(ios(13.0)) @interface UIWindowScene : UIScene
                                                     ^
    /Users/apple/work/workspace/code/flutter_app/piico_app/build/ios/Debug-iphonesimulator/LiveChat/LiveChat.framework/Headers/LiveChat-Swift.h:218:1: note: annotate 'setWindowScene:' with an availability attribute to silence this warning
    + (void)setWindowScene:(UIWindowScene * _Nullable)value;
    ^
    While building module 'LiveChat' imported from /Users/apple/work/flutterSDK/flutter/.pub-cache/hosted/pub.flutter-io.cn/livechatt-1.0.1+3/ios/Classes/LivechatPlugin.m:3:
    In file included from <module-includes>:2:
    /Users/apple/work/workspace/code/flutter_app/piico_app/build/ios/Debug-iphonesimulator/LiveChat/livechat.framework/Headers/LiveChat-Swift.h:217:4: warning: 'UIWindowScene' is only available on iOS 13.0 or newer [-Wunguarded-availability-new]
    + (UIWindowScene * _Nullable)windowScene SWIFT_WARN_UNUSED_RESULT;
       ^
    In module 'UIKit' imported from /Users/apple/work/workspace/code/flutter_app/piico_app/ios/Pods/Target Support Files/LiveChat/LiveChat-umbrella.h:2:
    /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator14.0.sdk/System/Library/Frameworks/UIKit.framework/Headers/UIWindowScene.h:17:50: note: 'UIWindowScene' has been marked as being introduced in iOS 13.0 here, but the deployment target is iOS 9.0.0
    UIKIT_EXTERN API_AVAILABLE(ios(13.0)) @interface UIWindowScene : UIScene
                                                     ^
    While building module 'LiveChat' imported from /Users/apple/work/flutterSDK/flutter/.pub-cache/hosted/pub.flutter-io.cn/livechatt-1.0.1+3/ios/Classes/LivechatPlugin.m:3:
    In file included from <module-includes>:2:
    /Users/apple/work/workspace/code/flutter_app/piico_app/build/ios/Debug-iphonesimulator/LiveChat/livechat.framework/Headers/LiveChat-Swift.h:217:1: note: annotate 'windowScene' with an availability attribute to silence this warning
    + (UIWindowScene * _Nullable)windowScene SWIFT_WARN_UNUSED_RESULT;
    ^
                                                                      API_AVAILABLE(ios(13.0))
    /Users/apple/work/workspace/code/flutter_app/piico_app/build/ios/Debug-iphonesimulator/LiveChat/livechat.framework/Headers/LiveChat-Swift.h:218:25: warning: 'UIWindowScene' is only available on iOS 13.0 or newer [-Wunguarded-availability-new]
    + (void)setWindowScene:(UIWindowScene * _Nullable)value;
                            ^
    In module 'UIKit' imported from /Users/apple/work/workspace/code/flutter_app/piico_app/ios/Pods/Target Support Files/LiveChat/LiveChat-umbrella.h:2:
    /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator14.0.sdk/System/Library/Frameworks/UIKit.framework/Headers/UIWindowScene.h:17:50: note: 'UIWindowScene' has been marked as being introduced in iOS 13.0 here, but the deployment target is iOS 9.0.0
    UIKIT_EXTERN API_AVAILABLE(ios(13.0)) @interface UIWindowScene : UIScene
                                                     ^
    While building module 'LiveChat' imported from /Users/apple/work/flutterSDK/flutter/.pub-cache/hosted/pub.flutter-io.cn/livechatt-1.0.1+3/ios/Classes/LivechatPlugin.m:3:
    In file included from <module-includes>:2:
    /Users/apple/work/workspace/code/flutter_app/piico_app/build/ios/Debug-iphonesimulator/LiveChat/livechat.framework/Headers/LiveChat-Swift.h:218:1: note: annotate 'setWindowScene:' with an availability attribute to silence this warning
    + (void)setWindowScene:(UIWindowScene * _Nullable)value;
    ^
                                                            API_AVAILABLE(ios(13.0))
    2 warnings generated.
    /Users/apple/work/flutterSDK/flutter/.pub-cache/hosted/pub.flutter-io.cn/livechatt-1.0.1+3/ios/Classes/LivechatPlugin.m:13:4: error: unknown receiver 'SwiftLivechatPlugin'; did you mean 'LivechatPlugin'?
      [SwiftLivechatPlugin registerWithRegistrar:registrar];
       ^~~~~~~~~~~~~~~~~~~
       LivechatPlugin
    In file included from /Users/apple/work/flutterSDK/flutter/.pub-cache/hosted/pub.flutter-io.cn/livechatt-1.0.1+3/ios/Classes/LivechatPlugin.m:1:
    /Users/apple/work/flutterSDK/flutter/.pub-cache/hosted/pub.flutter-io.cn/livechatt-1.0.1+3/ios/Classes/LivechatPlugin.h:3:12: note: 'LivechatPlugin' declared here
    @interface LivechatPlugin : NSObject<FlutterPlugin>
               ^
    2 warnings and 1 error generated.`

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -2,6 +2,16 @@
 
 A livechat widget for your mobile application.
 
+## 1.0.1+3
+
+* Fixed No podspec found on iOS
+
+## 1.0.1+2
+
+## 1.0.1+1
+
+* Fixed issue with plugin name
+
 ## 1.0.1
 
 * Fixed conflict with pods on ios

--- a/README.md
+++ b/README.md
@@ -2,7 +2,7 @@
 
 ![CI](https://github.com/mastersam07/livechat/workflows/CI/badge.svg?style=flat-square)
 [![license](https://img.shields.io/badge/license-MIT-success.svg?style=flat-square)](https://github.com/Mastersam07/livechat/blob/master/LICENSE)
-[![pub package](https://img.shields.io/pub/v/livechat.svg?color=success&style=flat-square)](https://pub.dartlang.org/packages/livechat)
+[![pub package](https://img.shields.io/pub/v/livechatt.svg?color=success&style=flat-square)](https://pub.dartlang.org/packages/livechatt)
 [![PRs Welcome](https://img.shields.io/badge/PRs-welcome-success.svg?style=flat-square)](https://github.com/Mastersam07/livechat/pulls)
 ![GitHub contributors](https://img.shields.io/github/contributors/mastersam07/livechat?color=success&style=flat-square)
 ![GitHub last commit](https://img.shields.io/github/last-commit/mastersam07/livechat?style=flat-square)
@@ -13,13 +13,13 @@ A livechat package for embedding mobile chat window in your mobile application.
 
 ```yaml
 dependencies:
-  livechat: "^1.0.1"
+  livechatt: "^1.0.1+3"
 ```
 
 ### ⚡️ Import
 
 ```dart
-import 'package:livechatinc/livechatinc.dart';
+import 'package:livechatt/livechatt.dart';
 ```
 
 ## 🎮 How To Use

--- a/android/src/main/java/tech/mastersam/livechat/LivechatPlugin.java
+++ b/android/src/main/java/tech/mastersam/livechat/LivechatPlugin.java
@@ -58,7 +58,7 @@ public class LivechatPlugin implements FlutterPlugin, MethodCallHandler, Activit
 
   private void onAttachedToEngine(Context applicationContext, BinaryMessenger messenger) {
     this.context = applicationContext;
-    channel = new MethodChannel(messenger, "livechatinc");
+    channel = new MethodChannel(messenger, "livechatt");
     channel.setMethodCallHandler(this);
   }
 

--- a/example/lib/main.dart
+++ b/example/lib/main.dart
@@ -2,7 +2,7 @@ import 'package:flutter/material.dart';
 import 'dart:async';
 
 import 'package:flutter/services.dart';
-import 'package:livechatinc/livechatinc.dart';
+import 'package:livechatt/livechatt.dart';
 
 void main() {
   runApp(MyApp());

--- a/example/pubspec.yaml
+++ b/example/pubspec.yaml
@@ -13,7 +13,7 @@ dependencies:
   flutter:
     sdk: flutter
 
-  livechatinc:
+  livechatt:
     # When depending on this package from a real application you should use:
     #   livechat: ^x.y.z
     # See https://dart.dev/tools/pub/dependencies#version-constraints

--- a/ios/Classes/SwiftLivechatPlugin.swift
+++ b/ios/Classes/SwiftLivechatPlugin.swift
@@ -4,7 +4,7 @@ import LiveChat
 
 public class SwiftLivechatPlugin: NSObject, FlutterPlugin {
   public static func register(with registrar: FlutterPluginRegistrar) {
-    let channel = FlutterMethodChannel(name: "livechatinc", binaryMessenger: registrar.messenger())
+    let channel = FlutterMethodChannel(name: "livechatt", binaryMessenger: registrar.messenger())
     let instance = SwiftLivechatPlugin()
     registrar.addMethodCallDelegate(instance, channel: channel)
   }

--- a/ios/livechatt.podspec
+++ b/ios/livechatt.podspec
@@ -3,8 +3,8 @@
 # Run `pod lib lint livechat.podspec' to validate before publishing.
 #
 Pod::Spec.new do |s|
-  s.name             = 'livechatinc'
-  s.version          = '1.0.1'
+  s.name             = 'livechatt'
+  s.version          = '1.0.1+3'
   s.summary          = 'A livechat package for embedding mobile chat window in your mobile application.'
   s.description      = <<-DESC
   A livechat package for embedding mobile chat window in your mobile application.

--- a/lib/livechatt.dart
+++ b/lib/livechatt.dart
@@ -3,7 +3,7 @@ import 'dart:async';
 import 'package:flutter/services.dart';
 
 class Livechat {
-  static const MethodChannel _channel = const MethodChannel('livechatinc');
+  static const MethodChannel _channel = const MethodChannel('livechatt');
 
   /// Get platform version
   static Future<String> get platformVersion async {

--- a/pubspec.yaml
+++ b/pubspec.yaml
@@ -1,6 +1,6 @@
-name: livechatinc
+name: livechatt
 description: A livechat package for embedding mobile chat window in your mobile application.
-version: 1.0.1
+version: 1.0.1+3
 homepage: https://github.com/Mastersam07/livechat
 repository: https://github.com/Mastersam07/livechat
 issue_tracker: https://github.com/Mastersam07/livechat/issues

--- a/test/livechatt.dart
+++ b/test/livechatt.dart
@@ -1,6 +1,6 @@
 import 'package:flutter/services.dart';
 import 'package:flutter_test/flutter_test.dart';
-import 'package:livechatinc/livechatinc.dart';
+import 'package:livechatt/livechatt.dart';
 
 void main() {
   const MethodChannel channel = MethodChannel('livechatinc');

